include/orbis/_types/save_data.h

```diff
@@ -60,21 +63,21 @@ typedef struct OrbisSaveDataMountInfo {
 	uint8_t reserved[32];
 } OrbisSaveDataMountInfo;
 
-typedef struct __attribute__((packed)) OrbisSaveDataMount
+typedef struct OrbisSaveDataMount
 {
 	int32_t userId;
-	int32_t unknown1;
-	const char *titleId;
-	const char *dirName;
-	const char *fingerprint;
-	uint64_t blocks;
-	uint32_t mountMode;
-	uint8_t reserved[36];
+	int32_t pad04;
+	const OrbisSaveDataTitleId*	titleId;
+	const OrbisSaveDataDirName*	dirName;
+	const OrbisSaveDataFingerprint*	fingerprint;
+	uint64_t 	blocks;
+	uint32_t 	mountMode;
+	uint8_t 	reserved[32];
```

Comment:
```suggestion
	uint8_t 	reserved[32];
	int32_t 	pad004;
```
If you're changing the reserved size, I think you should to add final `pad004` padding, to match struct size = 80 bytes, as defined in https://github.com/ChendoChap/Playstation-4-Save-Mounter/blob/master/structs.cs#L5